lib/facility_factory.rb

```diff
@@ -0,0 +1,58 @@
+class FacilityFactory
+
+  def create_facilities(facility_data)
+    facility_data.map do |facility|
+      Facility.new({
+        name:    format_name(facility),
+        address: format_address(facility),
+        phone:   format_phone(facility)
```

Comment:
Great job making these helper methods to format this data. Unfortunately, there is not good standardization across APIs so doing this kind of checking and formatting is often necessary.